nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1339,26 +1261,7 @@ private Charset getCharsetFromMediaType(MediaType contentType) {
      *
      * @return the directory in which the ETag cache should be written
      */
-    private static File getETagCacheDir() {
-        return Files.createTempDir();
-    }
-
-    private static class OverrideHostnameVerifier implements HostnameVerifier {
-
-        private final String trustedHostname;
-        private final HostnameVerifier delegate;
-
-        private OverrideHostnameVerifier(String trustedHostname, HostnameVerifier delegate) {
-            this.trustedHostname = trustedHostname;
-            this.delegate = delegate;
-        }
-
-        @Override
-        public boolean verify(String hostname, SSLSession session) {
-            if (trustedHostname.equalsIgnoreCase(hostname)) {
-                return true;
-            }
-            return delegate.verify(hostname, session);
-        }
+    private static File getETagCacheDir() throws IOException {
+        return Files.createTempDirectory(InvokeHTTP.class.getSimpleName()).toFile();
```

Comment:
Could it cause an issue in case of concurrent tasks?

Reply:
This method is called in the `setUpClient()` method which is annotated with `OnScheduled`.  This previous implementation did the same thing using Guava's Files class.  The approach seems acceptable since it is called when the OkHttpClient is being configured.